appveyor.yml

```diff
@@ -24,9 +24,16 @@ before_build:
 
 # scripts to run after build
 # ...testing
+# Note we do a ping, do save time if the test server is down.
+# Most of the rest of this is to avoid running tests on vendored dirs.
 after_build:
+  - keybase ping
   - cd ..
-  - go test -timeout 30m ./...
+  - go list ./... | find /V "vendor" > testlist.txt
```

Comment:
But maybe doing it here is better?  As I understand it, we're skipping it just because we don't have time on the CI, rather than because it shouldn't be run locally.